slack-post-msg/entrypoint.sh

```diff
@@ -1,19 +1,120 @@
 #!/bin/sh
-set -eu
+
+STATUS=${STATUS:-failure}
 
 if test -z "$SLACK_BOT_TOKEN"; then
   echo "Set the SLACK_BOT_TOKEN secret."
   exit 1
 fi
 
+if test -z "$CHANNEL_ID"; then
```

Comment:
```suggestion
if [[ -z $CHANNEL_ID ]]; then
```

If using Bash, can do this with an internal - no need to call `test`.